apps/arweave/src/ar_data_discovery.erl

```diff
@@ -105,16 +135,13 @@ handle_cast(update_network_data_map, #state{ peers_pending = N } = State)
 		{{value, Peer}, Queue} ->
 			monitor(process, spawn_link(
 				fun() ->
-					case ar_http_iface_client:get_sync_buckets(Peer) of
-						{ok, SyncBuckets} ->
-							gen_server:cast(?MODULE, {add_peer_sync_buckets, Peer,
-									SyncBuckets});
-						{error, request_type_not_found} ->
-							get_sync_buckets(Peer);
-						Error ->
-							?LOG_DEBUG([{event, failed_to_fetch_sync_buckets},
-								{peer, ar_util:format_peer(Peer)},
-								{reason, io_lib:format("~p", [Error])}])
+					fetch_sync_buckets(Peer),
```

Comment:
Do we want to disable the legacy syncing if `footprint_based_syncing` is enabled? My gut sense from the repack_in_place work is that having individual chunks repacked along side footprints can impact performance negatively - but perhaps the implications is different for syncing/packing.

Reply:
By "legacy" I mean the chunk-by-chunk syncing rather than the new footprint-by-footprint syncing. My sense from repack in place is that if we miss the two approaches within the same storage module it can lead to the chunk-by-chunk packing adding "random" chunks scattered throughout the partition that can mess with the efficiency of the footprint-by-footprint packing. So for repack-in-place I noticed better performance if I didn't mix the "chunk-by-chunk" approach with the "footprint-by-footprint" approach

This behavior may be unique to repack-in-place though (i.e. perhaps related to the logic it uses to determine if a given footprint needs to be repacked) so may be moot for sync/pack.